src/views/Register.vue

```diff
@@ -12,14 +12,13 @@
 
 <script>
 // @ is an alias to /src
-  import GitLogIn from '@/components/UsefulComponents/GitLogIn.vue'
+import GitLogIn from '@/components/UsefulComponents/GitLogIn.vue';
 
 export default {
   name: 'Editor',
   components: {
-    GitLogIn
+    GitLogIn,
```

Comment:
Really, a final comma at the end?

Reply:
Is deleted. Unfortunately prettier added automatically.

Reply:
Yeah, I also had some issues in the past in VSCode to get Prettier to respect the rules I set for TSLint/ESLint... :D Especially about those trailing commas...